Gems/Atom/RHI/DX12/Code/Source/RHI/Scope.cpp

```diff
@@ -317,19 +317,16 @@ namespace AZ
             AZ_PROFILE_FUNCTION(RHI);
 
             commandList.GetValidator().BeginScope(*this);
-
-            PIXBeginEvent(0xFFFF00FF, GetId().GetCStr());
-
-            if (RHI::Factory::Get().PixGpuEventsEnabled())
-            {
-                PIXBeginEvent(commandList.GetCommandList(), 0xFFFF00FF, GetId().GetCStr());
-            }
-
             commandList.SetAftermathEventMarker(GetId().GetCStr());
             
             const bool isPrologue = commandListIndex == 0;
             if (isPrologue)
             {
+                if (RHI::Factory::Get().PixGpuEventsEnabled())
+                {
+                    PIXBeginEvent(commandList.GetCommandList(), PIX_MARKER_CMDLIST_COL, GetMarkerLabel().data());
```

Comment:
nit: would be nice if each scope could have a different color

Reply:
Maybe something to add later. If we do add support for this it will need to be added to vk as well.